<ui version="4.0" >
 <author>Marek Krejza</author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>InfoMarksEditDlg</class>
 <widget class="QDialog" name="InfoMarksEditDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>392</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>4</hsizetype>
    <vsizetype>4</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>321</width>
    <height>392</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>321</width>
    <height>392</height>
   </size>
  </property>
  <property name="baseSize" >
   <size>
    <width>404</width>
    <height>620</height>
   </size>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle" >
   <string>Edit Markers</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QPushButton" name="objectModify" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>110</x>
        <y>340</y>
        <width>71</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text" >
       <string>Modify</string>
      </property>
     </widget>
     <widget class="QPushButton" name="objectCreate" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>340</y>
        <width>71</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text" >
       <string>Create</string>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel1_2_2" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>0</y>
        <width>202</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Objects near selected position:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="objectType" >
      <property name="geometry" >
       <rect>
        <x>100</x>
        <y>60</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text" >
        <string>Text mark</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Line</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Arrow</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="objectNameStr" >
      <property name="geometry" >
       <rect>
        <x>100</x>
        <y>90</y>
        <width>181</width>
        <height>18</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel1_2" >
      <property name="geometry" >
       <rect>
        <x>30</x>
        <y>60</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Object type:</string>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel2_2_3" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>90</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Object name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel2_2" >
      <property name="geometry" >
       <rect>
        <x>30</x>
        <y>120</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Text:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="objectsList" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>20</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel3_2" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>150</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Position:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="objectText" >
      <property name="geometry" >
       <rect>
        <x>100</x>
        <y>120</y>
        <width>181</width>
        <height>18</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="startLabel" >
      <property name="geometry" >
       <rect>
        <x>40</x>
        <y>170</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Start</string>
      </property>
     </widget>
     <widget class="QLabel" name="endLabel" >
      <property name="geometry" >
       <rect>
        <x>150</x>
        <y>170</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>End</string>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel4_3_4" >
      <property name="geometry" >
       <rect>
        <x>240</x>
        <y>170</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Layer</string>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel4_3_2" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>220</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Y1</string>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel4_3" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>190</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>X1</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="m_x1" >
      <property name="geometry" >
       <rect>
        <x>40</x>
        <y>190</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum" >
       <double>1e+10</double>
      </property>
      <property name="minimum" >
       <double>-1e+09</double>
      </property>
      <property name="singleStep" >
       <double>0.01</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="m_y1" >
      <property name="geometry" >
       <rect>
        <x>40</x>
        <y>220</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum" >
       <double>1e+10</double>
      </property>
      <property name="minimum" >
       <double>-1e+09</double>
      </property>
      <property name="singleStep" >
       <double>0.01</double>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel4_3_3" >
      <property name="geometry" >
       <rect>
        <x>130</x>
        <y>190</y>
        <width>24</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>X2</string>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel4_3_2_2" >
      <property name="geometry" >
       <rect>
        <x>130</x>
        <y>220</y>
        <width>24</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Y2</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="m_x2" >
      <property name="geometry" >
       <rect>
        <x>150</x>
        <y>190</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum" >
       <double>1e+10</double>
      </property>
      <property name="minimum" >
       <double>-1e+09</double>
      </property>
      <property name="singleStep" >
       <double>0.01</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="m_y2" >
      <property name="geometry" >
       <rect>
        <x>150</x>
        <y>220</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum" >
       <double>1e+10</double>
      </property>
      <property name="minimum" >
       <double>-1e+09</double>
      </property>
      <property name="singleStep" >
       <double>0.01</double>
      </property>
     </widget>
     <widget class="QSpinBox" name="m_layer" >
      <property name="geometry" >
       <rect>
        <x>240</x>
        <y>190</y>
        <width>44</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum" >
       <number>10</number>
      </property>
      <property name="minimum" >
       <number>-10</number>
      </property>
     </widget>
     <widget class="QToolButton" name="layerNButton" >
      <property name="geometry" >
       <rect>
        <x>140</x>
        <y>260</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>N</string>
      </property>
     </widget>
     <widget class="QToolButton" name="layerUpButton" >
      <property name="geometry" >
       <rect>
        <x>160</x>
        <y>260</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>U</string>
      </property>
     </widget>
     <widget class="QToolButton" name="layerEButton" >
      <property name="geometry" >
       <rect>
        <x>160</x>
        <y>280</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>E</string>
      </property>
     </widget>
     <widget class="QToolButton" name="layerWButton" >
      <property name="geometry" >
       <rect>
        <x>120</x>
        <y>280</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>W</string>
      </property>
     </widget>
     <widget class="QToolButton" name="layerDownButton" >
      <property name="geometry" >
       <rect>
        <x>120</x>
        <y>300</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>D</string>
      </property>
     </widget>
     <widget class="QToolButton" name="layerSButton" >
      <property name="geometry" >
       <rect>
        <x>140</x>
        <y>300</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>S</string>
      </property>
     </widget>
     <widget class="QPushButton" name="objectDelete" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>200</x>
        <y>340</y>
        <width>71</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text" >
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="objectDeleteAll" >
      <property name="enabled" >
       <bool>true</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>210</x>
        <y>270</y>
        <width>61</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text" >
       <string>Delete All</string>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel4_3_5" >
      <property name="geometry" >
       <rect>
        <x>30</x>
        <y>260</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Move selection:</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
